fnet/cli/predict.py

```diff
@@ -156,7 +156,7 @@ def parse_model(model_str: str) -> Dict:
 
 
 def save_predictions_csv(
-        path_csv: Path,
+        path_csv: str,
```

Comment:
Curious, we went from a path type to a string? Any reason?

Reply:
It seems like it got changed to Path for the associated utilities, i.e. path.exists()